view/frontend/templates/checkout/minicart_addons.phtml

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Bolt\Boltpay\ViewModel\MinicartAddons $miniCartView */
+$miniCartView = $block->getViewModel();
+if (!$miniCartView->shouldShow()) return;
+?>
+<script>
+    require([
+        'jquery', 'Magento_Customer/js/model/customer',
+    ], function ($, customer) {
+        function append_minicart_addons () {
+            require(['uiLayout'], function (layout) {
+                layout(<?php echo $miniCartView->getLayoutJSON(); ?>);
+
+                <?php if($miniCartView->configHelper->displayRewardPointsInMinicartConfig()):?>
+                $('#minicart-content-wrapper').on('click', '.totals.rewardpoints .action.delete', function (e) {
+                    e.preventDefault();
+                    $.ajax({
+                        url:        window.checkoutConfig.review.reward.removeUrl,
+                        type:       'GET',
+                        cache:      false,
+                        dataType:   'json',
+                        showLoader: true,
+                        success:    function (response) {
+                            $(document).trigger('bolt:createOrder');
+                            require(
+                                [
+                                    'Magento_Checkout/js/action/get-payment-information',
+                                    'Magento_Customer/js/customer-data'
+                                ],
+                                function (getPaymentInformationAction, customerData) {
+                                    getPaymentInformationAction().always(function () {
+                                        customerData.reload(['messages'], true);
+                                    });
+                                }
+                            );
+                        }
+                    });
+                });
+                <?php endif; ?>
+            });
+        }
+
+        if (typeof window.checkoutConfig == 'undefined') {
+            $.ajax({
```

Comment:
NOTE: it seems that jQuery `$.get` is configured in Magento to not cache the responses. Anyway, this is the safe way.

Reply:
thanks.  That's great to know.

Reply:
btw, @danac-gs , can you post some documentation or a reference that confirms this?  I'm having trouble finding this.

Reply:
Nevermind.  I found confirmation: